guides/getting-started.md

```diff
@@ -189,14 +189,14 @@ function populateDropdowns() {
 }
 
 function calculateCurrency() {
-  let initialAmount = $w("#sourceAmount").value;
+  let initialAmount = Number($w("#sourceAmount").value);
```

Comment:
I remember a discussion about converting this string to a number when we created the example. I think JavaScript automatically converts it to a number when it's included in a calculation, and we decided not to explicitly convert it so as not to add any more complexity to the code. Was the code not working unless you converted it?